upgrade/v1beta1/upgrade.go

```diff
@@ -100,17 +104,26 @@ func upgradeVolumes(namespace string, lhClient *lhclientset.Clientset) error {
 			return errors.Wrap(err, "failed to copy volume Status.CloneStatus")
 		}
 
-		_, err = lhClient.LonghornV1beta2().Volumes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
+		obj, err := lhClient.LonghornV1beta2().Volumes(namespace).UpdateStatus(context.TODO(), &new, metav1.UpdateOptions{})
 		if err != nil {
 			return errors.Wrapf(err, "failed to update for %v status %v", old.Kind, old.Name)
 		}
+
+		if err := tagCRLabelCRDAPIversion(obj); err != nil {
+			return errors.Wrapf(err, "failed to add label to %v %v", old.Kind, old.Name)
+		}
+		if _, err := lhClient.LonghornV1beta2().Volumes(namespace).Update(context.TODO(), obj, metav1.UpdateOptions{}); err != nil {
+			return errors.Wrapf(err, "failed to update for %v %v", old.Kind, old.Name)
+		}
```

Comment:
nit: suggestion to use `new.Kind` and `new.Name` rather than `old.Kind` and `old.Name`.